### PR TITLE
Short-circuit Librato when config is not provided

### PR DESCRIPTION
**Pull request by jayzes**
Rails's `empty?` would be nice here but I don't want to rely on it since this may not always be used in a Rails context.

### CODE REVIEW

File: lib/stream_reader/librato_reporter.rb [hunk 1]
```diff
@@ -17,6 +18,13 @@ def run!
     end
 
     private
+    
+    def config_present?
+      !ENV['LIBRATO_USER'].nil? &&
```
Comment (Justin-Holmes) [inline lib/stream_reader/librato_reporter.rb]: Would this ever be `nil` with the bang in front?

Comment (jayzes) [reply lib/stream_reader/librato_reporter.rb]: Good call - I think the `!` gets called last but it doesn't hurt to be explicit. Fixed in https://github.com/ello/kinesis-stream-reader/pull/19/commits/8f1f4a8be9b8361c81dbd46ba3746fa099c0fe59

File: lib/stream_reader/librato_reporter.rb [hunk 2]
```diff
@@ -17,6 +18,13 @@ def run!
     end
 
     private
+    
+    def config_present?
+      !(ENV['LIBRATO_USER'].nil?) &&
+        !(ENV['LIBRATO_USER'].empty?) &&
+        !(ENV['LIBRATO_TOKEN'].nil? &&
```
Comment (Justin-Holmes) [inline lib/stream_reader/librato_reporter.rb]: Might be missing a `)` here.

**Review by Justin-Holmes — APPROVED** [review]

File: lib/stream_reader/librato_reporter.rb (hunk 2 above)
Comment (jayzes) [reply lib/stream_reader/librato_reporter.rb]: D'oh. That's what I get for using Github.com to make a PR without syntax highlighting.

**Pull request merged**